app/src/main/java/com/simply/birthdayapp/main/addEvent/data/repository/BirthdayRepositoryImpl.kt

```diff
@@ -59,4 +62,72 @@ class BirthdayRepositoryImpl(
     }.catch { e ->
         emit(Result.Error(e.message ?: "", Birthday()))
     }.flowOn(Dispatchers.IO)
+
+    override fun updateBirthday(id: Int, input: CreateBirthdayInputDomain): Flow<Result<Birthday>> = flow {
+        emit(Result.Loading(Birthday()))
```

Comment:
Why do we need to pass Birthday in Loading state? Isn't it just showing a dummy loading?